[2023-03-29 15:24:49,701] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:24:48.652075+00:00 [queued]>
[2023-03-29 15:24:49,711] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:24:48.652075+00:00 [queued]>
[2023-03-29 15:24:49,711] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:24:49,711] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:24:49,711] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:24:49,725] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 15:24:48.652075+00:00
[2023-03-29 15:24:49,729] {standard_task_runner.py:52} INFO - Started process 83 to run task
[2023-03-29 15:24:49,731] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'manual__2023-03-29T15:24:48.652075+00:00', '--job-id', '419', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmp63wce985', '--error-file', '/tmp/tmpjg174qak']
[2023-03-29 15:24:49,732] {standard_task_runner.py:80} INFO - Job 419: Subtask start_task
[2023-03-29 15:24:49,782] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:24:48.652075+00:00 [running]> on host 75e36b0b32e7
[2023-03-29 15:24:49,853] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:24:48.652075+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:24:48.652075+00:00
[2023-03-29 15:24:49,854] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 15:24:49,854] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 15:24:49,880] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T152448, start_date=20230329T152449, end_date=20230329T152449
[2023-03-29 15:24:49,904] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:24:49,947] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
